README.md

```diff
@@ -73,6 +75,9 @@ Recommended setup for running as an app on Aptible:
      * `S3_BUCKET_BASE_PATH`: Destination path within bucket (Optional)
      * `S3_ACCESS_KEY_ID`: The access key you generated in step 3.
      * `S3_SECRET_ACCESS_KEY`: The secret key you generated in step 3.
+     * `INDEX_ARRAY`: array of indices, INDEX_ARRAY="logstash nginx", separated with white space 
```

Comment:
This should explain how these indice names should be formatted. `logstash` indices have a format we can reasonably rely on to extract the date, but that's not the case with arbitrary indices.